code/style.css

```diff
@@ -1,4 +1,285 @@
+
+.nav-toggle-label span,
+.nav-toggle-label::before,
+.nav-toggle-label::after {
+  display: block;
+  background: black;
+  height: 1px;
+  width: 1.5em;
+  border-radius: 5px;
+  position: relative;
+}
+
+.nav-toggle-label::before, 
+.nav-toggle-label::after {
+  content: ""; /* setting content to nothing for pseudoclasses */
+  position: absolute; 
+}
+
+.nav-toggle-label::before {
+  bottom: 50px;
+}
+
+.nav-toggle-label::after {
+  top: 50px;
+}
+
+.space{
+  height: 80px;
+}
+
+/* End of navbar
+
+  Section big news starts */
+
+.hero {
+  background: url(./Thailand.jpg);
+  background-position: center;
+  background-size: cover;
+  height: 40vh;
+}
+
+.big-news {
+  display: grid;
+  grid-row-gap: 10px;
+  padding: 20px 20px; /* 20px top and bottom 20px left and right */
+  text-align: center;
+  min-height: 30vh; /* for responsiveness */
+}
+
+.big-news h1 {
+  font-size: 30px;
+}
+
+.big-news p {
+  font-size: 20px;
+  padding: 0 30px;
+}
+
+.big-news img {
+  width: 100%;
+  height: 100%;
+  box-shadow: 0 4px 8px 0 rgba(206, 171, 171, 0.4), 0 6px 20px 0 rgba(0, 0, 0, 0.4);
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+.text {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+  padding: 0 20px 20px;
+  text-align: center;
+}
+
+.text a {
+  text-decoration: none;
+  font-size: 18px;
+  color: #111;
+  font-weight: 500;
+}
+
+.news-cards {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-column: span 1;
+  grid-column-gap: 40px;
+  align-items: stretch;
+  grid-row-gap: 40px;
+  margin: 20px 20px;
+}
+
+.card {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.4), 0 6px 20px 0 rgba(0, 0, 0, 0.4);
+}
+
+.card img {
+max-width: 100%;
+}
+
+footer {
+  display: grid;
+  grid-template-columns: 1fr, minmax(600px,1fr), 1fr;
+  color: rgb(136, 124, 124);
+  font-size: 14px;
+  padding: 30px 0;
+}
+
+footer p {
+  grid-column: 2/3;
+}
+
+footer span {
+  color: red;
+}
+
+/* this is for desktop */
+
+@media screen and (min-width: 901px) {
+  .nav-toggle-label {
+    display: none;
+  }
+
+  header {
+    display: grid;
+    grid-template-columns: .5fr auto minmax(600px, 1fr);
+  }
+
+  .logo {
+    display: flex;
+    align-items: center;
+    grid-column: 1/2;
+  }
+
+  .logo-text{
+    display: block;
+  }
+
+  nav {
+    color: black;
+    all:unset;
+    grid-column: 3/4;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+  }
+  nav ul {
+    display: flex;
+  }
+  nav li {
+    margin-right: 3em;
+    margin-bottom: 0;
+  }
+
+  nav a {
+    opacity: 1;
+    font-weight: 500;
+  }
+
+  .big-news {
+    grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+  }
+
+  .news-cards {
+    grid-template-columns: repeat(4, minmax(200px, 1fr));
+  }
+}
+
+/* this is for tablet */
+@media screen and (max-width: 900px) and (min-width: 600px) {
+
+  .big-news {
+    grid-template-columns: 1fr;
+  }
+  .news-cards {
+    grid-template-columns: 1fr 1fr;
+
+  }
+}
+
```

Comment:
You have a bracket on row 275 and one at 280 that might be "left overs"? Looks like it is double of them. 

Nicely work with all the comments and easy to read code. And everything works :)

Reply:
The bracket on row 275 closes the big news on row 273 and the bracket on row 280 closes the media screen on row 271.